packages/teleport/src/config.ts

```diff
@@ -65,7 +65,7 @@ const cfg = {
     consoleNodes: '/web/cluster/:clusterId/console/nodes',
     consoleConnect: '/web/cluster/:clusterId/console/node/:serverId/:login',
     consoleSession: '/web/cluster/:clusterId/console/session/:sid',
-    player: '/web/cluster/:clusterId/session/:sid',
+    player: '/web/cluster/:clusterId/session/:sid/:recordingType', // recordingType is one of 'ssh' | 'desktop'
```

Comment:
Recording type seems like it would make more sense as a query param than part of the path, but I don't _really_ care if this is how we typically do things.

Reply:
True, I was just going with what I know but its better to do it that way, especially since we plan to add the time parameter as well.